### PR TITLE
Replace npm/yarn/pnpm with npm2yarn

### PR DESCRIPTION
**Pull request by matevz**
Implements #502 .

### CODE REVIEW

File: docs/dapp/sapphire/quickstart.mdx [hunk 1]
```diff
@@ -154,8 +154,8 @@ Pretty cool if you ask me!
 
 Anyway, make it happen by running
 
-```sh
-PRIVATE_KEY="0x..." pnpm hardhat run scripts/run-vigil.ts --network sapphire_testnet
+```shell
```
Comment (aefhm) [inline docs/dapp/sapphire/quickstart.mdx:157]: Oof. We are not 100% consistent in this repo. 🤷‍♂️❓

File: docs/dapp/sapphire/quickstart.mdx [hunk 2]
```diff
@@ -154,8 +154,8 @@ Pretty cool if you ask me!
...
+PRIVATE_KEY="0x..." npx hardhat run scripts/run-vigil.ts --network sapphire_testnet
```
Comment (aefhm) [inline docs/dapp/sapphire/quickstart.mdx:158]: 👍

File: docs/general/images/architecture/technology_scalability.png
Comment (aefhm) [inline docs/general/images/architecture/technology_scalability.png:1]: Are we going to the PNGs? Just wondering if we are actively specifying the fallbacks.

**Review by aefhm — APPROVED** [review]

File: docs/general/images/architecture/technology_scalability.png
Comment (matevz) [reply docs/general/images/architecture/technology_scalability.png:1]: Oops, sorry, leftovers.

File: docs/dapp/sapphire/quickstart.mdx (hunk 1 above)
Comment (matevz) [reply docs/dapp/sapphire/quickstart.mdx:157]: Good call, I made those consistent now.

**Pull request merged**